openedx/core/djangoapps/agreements/models.py

```diff
@@ -28,9 +28,12 @@ class LTIPIISignature(models.Model):
     """
     This model stores a user's acknowledgement to share PII via LTI tools in a particular course.
     """
+    user = models.ForeignKey(User, db_index=True, on_delete=models.CASCADE)
     course_key = CourseKeyField(max_length=255, db_index=True)
-    user_id = models.IntegerField()
-    lti_tools_hash = models.IntegerField()
+    lti_tools = models.JSONField()
```

Comment:
I chose to add a field for the list of lti tools, because it may be helpful to see in addition to the hash. Are there any further thoughts about this?